### ISSUE TITLE
CSLA ProjectTracker project for MVC ASP.Net / Core / Razor?

### ISSUE THREAD

**Issue by grahamblandford** [MarimerLLC/cslaforum#538]
Hi guys,

I'm only now getting involved in Web development and venturing into the MVC/ASP fold.
I've been bouncing around looking at the different options and still trying to establish the best way forward.
I am typically a database developer with lots of CRUD-like interactions and have extensively used earlier versions of CSLA to develop very robust windows forms desktop applications - which are all still in production today (Thanks Rocky!)

My question comes down to, are there any online examples of MVC/ASP.Net/Core or Razor development available - something like the ProjectTracker application that was always a great source?

The Github links seem to be dead for "Getting Started"

I believe I still own the CSLA Business Objects book, but pretty sure that focused on WPF or webforms.

Any comments or suggestions would be appreciated!

Thanks,
Graham

**Comment by grahamblandford** [comment]
Thanks for that. It looks familiar, but I'll take a look.
To be honest, I'm trying to upgrade my skill set having found myself out of work last year. I think I'd like to go through the rigmarole of understanding the workings of the techniques in more detail rather than finding a "convert this" solution. 

My client that has been using my Windows forms application for the last 10+ years is doing more and more web work, and I'm involved in some of those projects. I'd like to leverage the knowledge I'm acquiring and use it to produce a web alternative to what I'd previously developed - especially as time goes by the more chance that Windows OS won't support the client ...

Thanks!
Graham

**Comment by fujiiface** [comment]
The CSLA 4 ebooks have a whole “book” dedicated to ASP.NET MVC.  There’s also the CSLA MVC sample on the CSLA repo that you could take a look at.  

https://github.com/MarimerLLC/csla/tree/master/Samples/NET/cs/CslaMVC

**Comment by tfreitasleal** [comment]
Just so you can see what I'm referring to, now you can try [Project Tracker online](http://projecttracker.ddns.net).

**Comment by grahamblandford** [comment]
Thanks fujiiface! That was the resource I was trying to find, but the link from Git was dead!

Tfreitasleal,
I can certainly understand the benefits of that for some new projects, and even porting the comprehensive desktop application I wrote, But the reality is, that kind of work hasn't fallen in my lap for some time, and most opportunities I have seen within the last year or so, have required significant web exposure; full stack and many if not all of those other technologies you mentioned.

**Comment by grahamblandford** [comment]
fujiiface,
do you have a live link to the eBooks? again, the links I have on git are dead

**Comment by rockfordlhotka** [comment]
Well it appears that my store has crashed - I'll try and figure out why... 😢

**Comment by fujiiface** [comment]
Sorry I’ve been busy today.  I was going to report the crashed store link but got side tracked 😔.  But it looks like Rocky got it back up.

**Comment by grahamblandford** [comment]
Hey guys,
I'm still having some issues cracking this new version of CSLA and finding the appropriate documentation for one of the current technologies for web development.

I've reviewed the eBooks that I had previously purchased, and also looked at the old MV3UI project that was released around that time and it's just added more questions than answers. That particular project seems to combine MVC AND Razor, whereas all of the new microsoft learning that I've been undertaking seems to suggest that MVC5 and Razor are two separate techniques.

I also looked at the Customer/Orders project that was floating around and that one won't compile either.

I managed to get the current Project tracker running with WPF and also as WinForm, but I'd really like a clear push into either the MVC or Razor direction, and as I'm new to development in both areas - I could really do with a concise example rather than "apply these rules to your MVC/razor project".

Any ideas or suggestions are welcome.

Thanks!
Graham
